### ISSUE TITLE
[Bug]: Can't pick up items or collect xp

### ISSUE THREAD

**Issue by manicmasen** [someaddons/performant_issues#232]
### Describe the bug you're experiencing

Bug: In a single-player world, whenever any item is thrown or dropped (like from mobs), and if any xp is spawned, it cannot be collected by the player. 

However, when restarting the game, the dropped items that are still there can be picked up after restart. No new drops or xp can be picked up again in the current play session until another restart. 

There were no changes made to the config file, and removing Performant from the pack resolved the issue. I'm using Performant version 1.16.2-5-3.92m.

What I expected to happen: Items and xp to be auto-collected. 

### Reproducability

Bug can be reproduced at any time by any action that results in a dropped item or xp. 

### Mod up to date

- [X] Before submitting this issue I updated to the newest version and reproduced it

### Minecraft version

1.16.5

### Modloader version

Forge

### Logs

https://gist.github.com/manicmasen/5b99ed62202f5b0027a6e216745ed95d

**Comment by someaddons** [comment]
I'm unable to reproduce this with performant + vanilla, must have some conflicts with other mods you're running

**Comment by manicmasen** [comment]
Interesting. Thank you for checking it out for me! I just wanted to make the issue known. But it makes sense that it is a conflict.

**Comment by someaddons** [comment]
may be related to https://github.com/ThePaul-T/WYML/issues/14

**Comment by manicmasen** [comment]
That does look like the same issue. What's weird is that I disabled WYML, but the issue still persisted as long as Performant was installed. However, that was in the same world that the issue originally began in. 
Later I will remove WYML and keep Performant, create a new world this time, and see if the issue continues. I'll comment back with the results.

**Comment by manicmasen** [comment]
So I made a new instance, with only Performant installed and it worked fine. Once I added WYML, the bug happened. 
Then I made a new instance with only WYML installed and it also worked fine. Once I added Performant, the bug happened. So yeah, it's definitely a specific conflict between the 2 mods. 

I looked around in the configs for both mods and made a few tweaks to see if it would resolve the bug, but none of my changes helped.
Thanks for finding that other issue link, it's at least good to know it's a mod conflict and not a bug in either of the actual mods!
